docs/getting-started/12-bind-redis-service-to-function.md

```diff
@@ -0,0 +1,212 @@
+
+   - The **spec.parameters.envPrefix.name** field is optional. On creating a ServiceBinding, it adds a prefix to all environment variables injected in a Secret to the Function. In our example, **envPrefix** is `REDIS_`, so all environment variables will follow the `REDIS_{env}` naming pattern.
+
+     > **TIP:** It is considered good practice to use **envPrefix**. In some cases, a Function must use several instances of a given ServiceClass. Prefixes allow you to distinguish between instances and make sure that one Secret does not overwrite another one.
+
+4. Check that the ServiceBindingUsage CR was created. Tis is indicated by the last condition in the CR status equal to `Ready True`:
+
+  ```bash
+  kubectl get servicebindingusage orders-function -n orders-service -o=jsonpath="{range .status.conditions[*]}{.type}{'\t'}{.status}{'\n'}{end}"
+  ```
+
+To see the Secret details and retrieve them from the ServiceBinding, run this command:
+
+  ```bash
+  kubectl get secret orders-function -n orders-service -o go-template='{{range $k,$v := .data}}{{printf "%s: " $k}}{{if not $v}}{{$v}}{{else}}{{$v | base64decode}}{{end}}{{"\n"}}{{end}}'
+  ```
+
+  Expect a response similar to this one:
+
+  ```bash
+  HOST: hb-redis-micro-0e965585-9699-443f-b987-38bc6af0e416-redis.serverless.svc.cluster.local
+  PORT: 6379
+  REDIS_PASSWORD: 1tvDcINZvp
+  ```
+
+  </details>
+  <details>
+  <summary label="console-ui">
+  Console UI
+  </summary>
+
+1. Go to **Development** > **Functions** in the left navigation panel and select `orders-function`.
+
+2. Switch to the **Configuration** tab and select **Create Service Binding** in the **Service Bindings** section.
+
+3. Select `redis-service` from the **Service Instance** drop-down list, add `REDIS_` as **Prefix for injected variables**, and make sure **Create new Secret** is selected.
+
+   > **NOTE:** The **Prefix for injected variables** field is optional. On creating a ServiceBinding, it adds a prefix to all environment variables injected in a Secret to the Function. In our example, the prefix is set to `REDIS_`, so all environment variables will follow the `REDIS_{ENVIRONMENT_VARIABLE}` naming pattern.
+
+   > **TIP:** It is considered good practice to use prefixes for environment variables. In some cases, a Function must use several instances of a given ServiceClass. Prefixes allow you to distinguish between instances and make sure that one Secret does not overwrite another.
+
+4. Select **Create** to confirm the changes.
+
+A message confirming that the ServiceBinding was created will appear in the **Service Bindings** section in your Function along with environment variable names.
```

Comment:
```suggestion
A message confirming that the ServiceBinding was created will appear in the **Service Bindings** section in your Function's view along with environment variable names.
```
I just thought, should we add `view` here? We don't really see it in the Function itself, right?

Reply:
You're right :)